infrastructure/aws/cdk/app.py

```diff
@@ -76,8 +74,9 @@ def __init__(
             timeout=Duration.seconds(timeout),
             environment={
                 **DEFAULT_ENV,
-                **environment,
                 "TITILER_CMR_ROOT_PATH": app_settings.root_path,
+                "TITILER_CMR_S3_AUTH_STRATEGY": app_settings.s3_auth_strategy,
+                "AWS_REQUEST_PAYER": app_settings.aws_request_payer,
```

Comment:
Should we only add this `AWS_REQUEST_PAYER` if `app_settings.aws_request_payer is not None`?